### ISSUE TITLE
Sync re-syncs every file when device adds metadata causing file size to increase

### ISSUE THREAD

**Issue by bdarfler** [gpodder/gpodder#1496]
👋🏻 I'm running 3.11.1 on MacOS 10.15.7 and syncing to a SanDisk Sport Go.

Since updating from 3.11.0 I have noticed that every sync re-syncs every file. This is incredibly tedious.

I wonder if it has something to do with https://github.com/gpodder/gpodder/pull/1434.

I'm happy to add more details or help debug in any way I can.

**Comment by auouymous** [comment]
Can you install https://app.circleci.com/pipelines/github/gpodder/gpodder/582/workflows/0d5fc8dc-c3b9-489d-8ac5-e6b0dad4cbd5/jobs/948/artifacts when it finishes building in a few minutes and then run gpodder with `-v` (https://gpodder.github.io/docs/macos.html#debugging-gpodder-on-macos). Check the logs after performing a sync and see if the sizes are different.

```
[gpodder.sync] INFO: Copying TEST_file.ogg (81393 bytes) => file:///home/test/sync-test/test%20feed/TEST_file.ogg (81393 bytes)
```

**Comment by bdarfler** [comment]
Can confirm the sizes are different.

`[gpodder.sync] INFO: Copying Birthday Parties vs. Trick-or-Treating.mp3 (33810228 bytes) =>
 file:///Volumes/SPORT%20GO/Podcasts/Smash%20Boom%20Best/Birthday%20Parties%20vs.%20Trick-or-Treating.mp3 (33846120 bytes)`

I'm attaching the [full logs](https://github.com/gpodder/gpodder/files/11072790/gpodder.log.zip) if that is helpful.

**Comment by auouymous** [comment]
Ok, the device is making changes (probably metadata) to the file, causing the size to increase. I'll look into reverting that patch, and adding another way to overwrite partially uploaded files.

**Comment by bdarfler** [comment]
Thanks!

**Comment by bdarfler** [comment]
Aww, I was hoping at least the revert would go out in the last release.

**Comment by auouymous** [comment]
Sorry about that, it is now fixed. You can download the fix at https://app.circleci.com/pipelines/github/gpodder/gpodder/609/workflows/41952eea-42ef-4cb8-beff-e33dbc7c8cab/jobs/972/artifacts or wait for 3.11.3 to be released in a week or two, after #1538 is fixed.

**Comment by bdarfler** [comment]
Thanks!
